src/modules/Kdump.rb

```diff
@@ -1145,7 +1145,11 @@ def crash_kernel_values
       # modification.
       # The old value (ensuring the Array format) will be returned.
       if @crashkernel_list_ranges
-        return Array(@crashkernel_param_values.dup)
+        if @crashkernel_param_values.is_a? Symbol
+          return Array(@crashkernel_param_values)
```

Comment:
Sorry for being so picky :-( but to fully match the method documentation, this should be `return Array(@crashkernel_param_value.to_s)`